.github/workflows/build.yml

```diff
@@ -43,6 +43,20 @@ jobs:
             cxx-compiler: clang++
             cc-compiler: clang
             compiler-desc: Clang
+          - name_prefix: Linux CentOS 7 VFX CY2023
+            release_prefix: linux-vfx2023
+            ostype: linux
+            distro: centos7
+            os: ubuntu-latest
+            container: aswf/ci-base:2023
+            vfx-cy: 2023
+            has_cmake_presets: false
+            buildtype: Release
+            conan_version: 2.0.4
```

Comment:
`aswf-docker` containers already have a version of Conan, currently 1.58.0, but it probably makes sense to use your own version across all builds, especially given the flux created by the transition from 1.x to 2.x